src/popup/router/components/Button.vue

```diff
@@ -46,6 +47,7 @@ export default {
     inactive: Boolean,
     to: [String, Object],
     bold: Boolean,
+    secondaryModal: Boolean,
```

Comment:
can it be combined with `dark`? I don't think that we need so lot of types of buttons

Reply:
`dark` is used in modals and a few pages that are not rebranded yet. 

In the design system modals have special secondary button, not sure if the same will be used on the pages using `dark` at the moment (designs not available).

Reply:
@onvisions can we make all old `dark` buttons to have the same color as in new designs?

Reply:
@davidyuk , yes we can. :) I synced the issue with @imollov.
The reason for different secondary button color in the modals was the contrast between modal background and button but I think we can use #232323 for all secondary buttons.